### ISSUE TITLE
Error: cron reached maximum iterations -- please stop reporting this bug

### ISSUE THREAD

**Issue by suntong** [kelektiv/node-cron#579]
**Stop!**

Most people comes here when they saw the error of 
`Error: Something went wrong. cron reached maximum iterations.`

And the ask of 

> Please open an issue (https://github.com/kelektiv/node-cron/issues/new) and provide the following string

But if you do a search, 
https://github.com/kelektiv/node-cron/issues?page=2&q=cron+reached+maximum+iterations
you'll find that the one opened on Feb 11, 2020 by shivam-postman is still unanswered. 

So don't waste your effort -- I found this out only after I found that I've send the same error twice, both followed up on somebody else's previous error report.

**Comment by mryellow** [comment]
> But if you do a search,

They're all good as closed mate, no one is working on this turd. 

The library is dead. It doesn't work and has gibberish all through `_getNextDateFrom`.

**Comment by intcreator** [comment]
closing in favor of #467
